build_tools/ci/cpu_comparison/run.py

```diff
@@ -1581,74 +1581,6 @@ def __init__(self):
         self.existing_names = []
         self.tests = []
 
-        # Tests Matmul + Trunci.
```

Comment:
Removing the matmul + trunci tests to reduce CI time as there are still matmul+scaling+trunci tests

Reply:
We should then cleanup the `matmul_trunci*` template test and the base class ?

Reply:
Done